l10n/configs/pontoon.toml

```diff
@@ -107,7 +101,6 @@ locales = [
     "zam",
     "zh-CN",
     "zh-TW",
-    "zu",
```

Comment:
`zu` is also in our `PROD_LANGUAGES` setting, should we remove it there as well?

Reply:
Good catch! Fixing now! ❤️